src/components/common/Filter.js

```diff
@@ -1,10 +1,11 @@
 import FilterListRoundedIcon from '@mui/icons-material/FilterListRounded';
+import Alert from '@mui/material/Alert';
 import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
 import Grid from '@mui/material/Grid';
 import TextField from '@mui/material/TextField';
 import Typography from '@mui/material/Typography';
-import log from 'loglevel';
+// import log from 'loglevel';
```

Comment:
this can just be deleted, imports can be easily replaced so there is no need to preserve them.
```suggestion
```